### DISCUSSION TITLE
[Feature Request] Memory View highlight recent changes

### DISCUSSION THREAD

**Discussion by Whithbrin355** [RetroAchievements/RAIntegration#1053] [Ideas]
I'd really like it if there was a way in memory view to toggle a setting so it will highlight values that change in any way.

I can't tell you the number of times I've been looking for a single bit flag in a particular block and I know it's there, but I just glance over it in a sea of all the same numbers. (I know 32-bit view can make it a bit easier to notice but it's still not perfect.)

**Comment by Jamiras** [comment]
What I usually do is is pause the emulator, do a search from 0xADDR1 to 0xADDR2, then filter on equals (everything should match). Then you can multi-select the search results and turn them into bookmarks. When you know something is about to happen, clear the bookmark change counters. Then after it happens, the entries that changed will have a non-zero change counter. You can even sort by Changes to more quickly find them.

I'm not saying that some sort of in-memory highlight is a bad idea - just illustrating a way to handle the problem now.

**Comment by Jamiras** [comment]
Also, this could get complicated by the other existing highlights. If something with a code note changes, should it be colored to indicate the change, or maintain the color indicating it has a code note attached?

**Comment by Whithbrin355** [comment]
That's true... Personally, I think it would be nice if the highlight doesn't actually change the color but just, you know, highlights it. Like an overlay color so you can still the color underneath it.

I don't know if that would be possible or not.

But if the highlight is a quick toggle that you can enable and disable then not maintaining the previous color wouldn't be too big of a deal I don't think.

**Comment by Whithbrin355** [comment]
I suppose another option might be to have a different color highlight if there's a code note.
